bundles/binding/org.openhab.binding.rfxcom.test/.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>org.openhab.binding.rfxcom.test</name>
+	<name>org.openhab.binding.rfxcom.test1</name>
```

Comment:
Shouldn't this be `org.openhab.binding.rfxcom1.test` ?

Reply:
That seems more logical indeed so I'll rename it.